source/views/sis/course-search/detail/index.js

```diff
@@ -139,6 +139,24 @@ function Schedule({course}: {course: CourseType}) {
 	)
 }
 
+function Notes({course}: {course: CourseType}) {
+	let notesText = course.notes ? course.notes.join(' ') : ''
```

Comment:
This should be joined on newline (same on the list view) IMO

Reply:
I'll leave it up to you, but I think spaces looks better.

<details>
<summary>Screenshots</summary>

Spaces | Newlines
--|--
![android-space2](https://user-images.githubusercontent.com/5240843/45123644-cd884900-b124-11e8-8d39-b8e50d0dc395.png) | ![android-newline2](https://user-images.githubusercontent.com/5240843/45123642-cd884900-b124-11e8-8337-fb0ce2c5618b.png)
![android-space](https://user-images.githubusercontent.com/5240843/45123643-cd884900-b124-11e8-807a-51cdf75484cc.png) | ![android-newline](https://user-images.githubusercontent.com/5240843/45123641-cd884900-b124-11e8-9e9f-ae66e1c97fa5.png)

</details>